test/relations.test.js

```diff
@@ -918,6 +918,51 @@ describe('relations', function() {
       };
     });
 
+    describe('find over related model with options', function() {
+      after(function() {
+        Physician.clearObservers('access');
+        Patient.clearObservers('access');
+      });
+      before(function() {
+        Physician.observe('access', beforeAccessFn);
+        Patient.observe('access', beforeAccessFn);
+
+        function beforeAccessFn(ctx, next) {
+          ctx.query.where.realm = ctx.options.realm;
+          next();
+        }
+      });
+      it('should find be filtered from option', function(done) {
+        var id;
+        Physician.create(function(err, physician) {
+          if (err) return done(err);
+          physician.patients.create({name: 'a', realm: 'test'}, function(err, ch) {
+            if (err) return done(err);
+            id = ch.id;
+            physician.patients.create({name: 'z', realm: 'test'}, function(err) {
+              if (err) return done(err);
+              physician.patients.create({name: 'c', realm: 'anotherRealm'}, function(err) {
+                if (err) return done(err);
+                verify(physician);
+              });
+            });
+          });
+        });
+
+        function verify(physician) {
+          physician.patients({}, {realm: 'test'}, function(err, records) {
```

Comment:
Can we specify an order filter i.e.`{order: 'ASC'}` here so that the actual values will always be in ascending order? This will help fix the CI failures.